addons/mail/static/src/core/web/recipient_list.js

```diff
@@ -22,6 +22,11 @@ export class RecipientList extends Component {
                 this.props.thread.loadMoreRecipients();
             }
         });
+        onMounted(() => {
+            if (this.props.load) {
+                this.props.load(this.props.thread, ["followers", "suggestedRecipients"]);
+            }
```

Comment:
Same remark but inverse: `suggestedRecipients` is relevant for this component, but not `followers`.

I would also avoid adding another prop that couples this component with form view / chatter code (i.e. parented components, which is not enforced by how the component is used). Instead I would make use of `loadMoreRecipients` so this is explicit dependance of core Discuss feature, which makes this component work in any context (i.e. can be mounted without a parented link with FormView/Chatter, and it just works).